packages/ckeditor5-editor-multi-root/src/multirooteditor.ts

```diff
@@ -637,6 +637,7 @@ export default class MultiRootEditor extends DataApiMixin( Editor ) {
 			resolve(
 				editor.initPlugins()
 					.then( () => editor.ui.init() )
+					.then( () => editor._verifyRootsWithInitialData() )
```

Comment:
Let's integrate it with the next promise so that we don't introduce the next microtask.

Reply:
Maybe add some comments about why here?